### PR TITLE
Add basic integration test

### PR DESCRIPTION
**Pull request by josephfrazier**
(following up on https://github.com/mantoni/eslint_d.js/pull/67#issuecomment-295379736)

This is adapted from the tests [here], and simply ensures that eslint_d
reformats an improperly formatted file, using the eslint configuration
of this project.

[here]: https://github.com/josephfrazier/prettier_d.js/blob/master/test.sh

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+file=./test/prettier.js
+
+$eslint stop
+$eslint start
+
+PORT=`cat ~/.eslint_d | cut -d" " -f1`
+TOKEN=`cat ~/.eslint_d | cut -d" " -f2`
+
+md5sum $file | grep 904306b7a51e4f634016cccea8924ea1 >/dev/null
+$eslint --fix-to-stdout --stdin < $file | md5sum | grep f75b2b44fd861a20b69f9a3e1960e419 >/dev/null
```
Comment (aaronjensen) [inline test.sh]: I'd rather this were diff'd against another file, it'd be easier to update the tests.

File: test.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euo pipefail # bash "strict mode"
```
Comment (aaronjensen) [inline test.sh]: 👍

File: package.json [hunk 3]
```diff
@@ -19,7 +19,9 @@
   ],
   "homepage": "https://github.com/mantoni/eslint_d.js",
   "scripts": {
-    "test": "eslint ."
+    "lint": "eslint bin lib",
```
Comment (aaronjensen) [inline package.json]: Probably should add the test directory to `.eslintignore` instead of this, better to be explicit about what not to lint.

**Comment by mantoni** [comment]
Cannot merge any more. I'll close this for now. Thanks everyone.

**Pull request closed without merging**